src/Symfony/Component/ObjectMapper/Tests/Fixtures/ClassMap/QuoteRequestView.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symfony\Component\ObjectMapper\Tests\Fixtures\ClassMap;
+
+use Symfony\Component\ObjectMapper\Attribute\Map;
+
+#[Map(source: Quote::class)]
+final class QuoteRequestView
+{
+    public string $id;
+
+    #[Map(target: CostRequestView::class)]
```

Comment:
```suggestion
```

Reply:
we should remove this, a target can not be a `class-string` when mapped to a property